### ISSUE TITLE
[Enhancement]: Add support for HID Consumer control Play/Pause

### ISSUE THREAD

**Issue by bigdoug2005** [advplyr/audiobookshelf#2423]
### Describe the feature/enhancement

**Describe the feature**
On iOS the Audiobookshelf app does not appear to react to the HID Consumer control play/pause toggle command 0xCD.  The individual play and pause commands work 0xB1 and 0xB0.

**Audiobookshelf version**
iOS 0.9.68-beta

**Comment by Sapd** [comment]
(The issue is in the wrong repo).

But Probably an easy fix:
https://github.com/advplyr/audiobookshelf-app/blob/793f0c05f7d54d6c95759b6b7daaa78c71e28478/ios/App/Shared/player/AudioPlayer.swift#L579-L593

Of `MPRemoteCommandCenter` play and pausecommand is implemented, however `togglePlayPause` has been forgotten: https://developer.apple.com/documentation/mediaplayer/mpremotecommandcenter/1618992-toggleplaypausecommand

**Comment by advplyr** [comment]
Is this a duplicate of https://github.com/advplyr/audiobookshelf-app/issues/622?

**Comment by KaiStarkk** [comment]
> Is this a duplicate of [advplyr/audiobookshelf-app#622](https://github.com/advplyr/audiobookshelf-app/issues/622)?

Duplicate as noted above. There is a proposed fix, just waiting on testing. I don’t have a macOS dev environment to test (virtualising is against their TOS on non Apple hardware) but am looking into it. Any other devs with Xcode welcome to take it

**Comment by advplyr** [comment]
Duplicate of https://github.com/advplyr/audiobookshelf-app/issues/622
